velox/functions/prestosql/StringFunctions.h

```diff
@@ -267,4 +267,77 @@ struct StrLPosFunction : public StrPosFunctionBase<T, true> {};
 template <typename T>
 struct StrRPosFunction : public StrPosFunctionBase<T, false> {};
 
+template <typename T>
+struct LevenshteinDistanceFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<int64_t>& result,
+      const arg_type<Varchar>& left,
+      const arg_type<Varchar>& right) {
+    doCall<false>(result, left, right);
+  }
+
+  FOLLY_ALWAYS_INLINE void callAscii(
+      out_type<int64_t>& result,
+      const arg_type<Varchar>& left,
+      const arg_type<Varchar>& right) {
+    doCall<true>(result, left, right);
+  }
+
+  template <bool isAscii>
+  FOLLY_ALWAYS_INLINE void doCall(
+      out_type<int64_t>& result,
+      const arg_type<Varchar>& left,
+      const arg_type<Varchar>& right) {
+    std::vector<int32_t> leftCodePoints =
+        stringImpl::stringToCodePoint<isAscii>(left);
+    std::vector<int32_t> rightCodePoints =
+        stringImpl::stringToCodePoint<isAscii>(right);
+    if (leftCodePoints.size() < rightCodePoints.size()) {
+      std::vector<int32_t> temp = leftCodePoints;
+      leftCodePoints = rightCodePoints;
+      rightCodePoints = temp;
+    }
+
+    if (rightCodePoints.size() == 0) {
+      result = leftCodePoints.size();
+      return;
+    }
+
+    const auto combinedInputLimit = 1000000;
```

Comment:
1'000'000 for readability

`static const int32_t kCombinedInputLimit = 1'000'000;`

Please, review coding style guidelines and update to conform.